### DISCUSSION TITLE
Gigabyte Z87: No ReBar after applying FFS

### DISCUSSION THREAD

**Discussion by vshah23** [xCuri0/ReBarUEFI#208] [General]
![image](https://github.com/xCuri0/ReBarUEFI/assets/6758788/b4245c56-bed0-4912-a8a5-c05776a82346)
![image](https://github.com/xCuri0/ReBarUEFI/assets/6758788/765c77c2-bc36-40e2-9a5d-428ba3525cf2)

Setup:
- OS: Windows 11
- Mobo: Gigabyte Z87N-WIFI
- GPU: AMD RX 6600XT
- BIOS: [F6](https://www.gigabyte.com/Motherboard/GA-Z87N-WIFI-rev-1x/support#support-dl-bios)

Things I tried:

Above 4G Decode:
- Followed the guide to enable hidden 4g decoding by booting grub shell efi from a spare Windows USB:
  - EDIT: Just checked the FAQs and realized that `Memory Remap` is [not ](https://github.com/xCuri0/ReBarUEFI/wiki/Common-issues-(and-fixes)#is-memory-remap-in-the-same-as-4g-decoding-) what I was supposed to be looking for...
  - EDIT 2: I'm not able to find the `Above 4G Decode` offset from the files that IFRExtractor outputs. I do see the following string in the hex view in UEFITool though:
    - `Above 4G DecodingEnables or Disables 64bit capable Devices to be Decoded in Above 4G Address Space (Only if System Supports 64 bit PCI Decoding).`

ReBar:
- First thing I tried before I found this repo was applying the registry edits for enabling ReBar on unsupported GPUs, but I now realize that probably doesn't apply to my hardware anyway. I left the registry edits in place as I continued with the next steps.
- Added FFS module to F6 BIOS from above hardware section
- Flashed BIOS with patched version
- CSM off
- Secure Boot off
- Played around with TOLUD (not sure if necessary or applicable, left it at 0xC for 3.25 setting for now, default was dynamic or 0x0):
  - <img src="https://github.com/xCuri0/ReBarUEFI/assets/6758788/3a172b24-c20e-4c3e-9708-ab6858c7b257" width="400">

**Comment by xCuri0** [comment]
Did you enable it with ReBarState ?

**Comment by vshah23** [comment]
That worked, thanks!

I misread the documentation and thought I was supposed to follow the ReBarState portion of the guide **after** seeing ReBar enabled in Adrenaline software.

Now I just have the issue of trying to enable `Above 4G Decode`, but I'm guessing that's the same issue as #14...

**Comment by vshah23** [comment]
Gigabyte told me my board is too old to support Above 4G Decoding when I asked them to provide BIOS with it enabled :(
